docker-compose-local.yml

```diff
@@ -1,8 +1,6 @@
 version: "3.3"
 services:
   iroha:
-    build:
-      context: .
     image: iroha:debug
```

Comment:
this compose file is for local build so these lines should not be deleted

Reply:
no prob, but isn't this the default?

Reply:
Did you get the same result?
At least I did not.
Without `build` it tries to pull `iroha:debug` image from the repository.
I think we have never published such a image

Reply:
These lines are usually needed if you were to put this `docker compose.yml` somewhere other than project root. I think it’s better to remove them for now.

Reply:
@s8sato I checked and you were correct, I'll restore this